crates/context_aware_config/src/api/context/helpers.rs

```diff
@@ -120,14 +123,40 @@ pub fn validate_condition_with_functions(
         .filter_map(|(key, f_name)| f_name.map(|func| (key, func)))
         .collect();
 
+    let context_validation_function = get_context_validation_function(conn, schema_name)?;
+
+    let environment = FunctionEnvironment {
+        context: context_map.clone(),
+        overrides: override_.clone(),
```

Comment:
should context validation require overrides as well ?
should this be at all supported ?

shouldn't it simply be, just like how it is being done for remote cohort validation:

```suggestion
        overrides: Map::new(),
```